handbook/information-security/statement-of-applicability.md

```diff
@@ -279,27 +279,17 @@ activities.
 
 ### A15.1 - Information security in supplier relationships
```

Comment:
Are we doing titlecase or sentence case for titles? I'd recommend titlecase

Reply:
These are the titles of the ISO27001 Annexes. I think we should leave them formatted as in the place we're quoting